openwisp_users/apps.py

```diff
@@ -23,16 +23,28 @@ class OpenwispUsersConfig(AppConfig):
     verbose_name = _('Users and Organizations')
 
     def ready(self):
-        self.register_default_menu_items()
+        self.register_menu_group()
         self.set_default_settings()
         self.connect_receivers()
 
-    def register_default_menu_items(self):
-        register_menu_items(
-            [
-                {'model': settings.AUTH_USER_MODEL},
-                {'model': get_model_name('openwisp_users', 'Organization')},
-            ]
+    def register_menu_group(self):
```

Comment:
Code wise it looks good, but try to run the app.
It didn't show Users and Organizations in my menu! :-(

Reply:
Icons were not added. I will add them.